src/System.Collections.NonGeneric/tests/SortedListTests.cs

```diff
@@ -1554,6 +1554,7 @@ public class SortedList_SyncRootTests
 
         [Fact]
         [OuterLoop]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)] // Changed behavior
```

Comment:
Nit: can you include a link to the PR that changed the behavior?  I believe SkipOnTargetFramework's optional second argument is also a string reason that can be used instead of a comment.